app/app.go

```diff
@@ -26,6 +26,7 @@ import (
 // and its only purpose is to provide business logic to Server via its methods.
 type App struct {
 	ch *Channels
+	ID string
```

Comment:
We should not do this. Find alternatives.

Reply:
Not able to find other alternatives